server/plugin.go

```diff
@@ -923,3 +971,21 @@ func (p *Plugin) OnSharedChannelsSyncMsg(msg *model.SyncMsg, _ *model.RemoteClus
 
 	return resp, nil
 }
+
+func (p *Plugin) ChannelHasBeenCreated(c *plugin.Context, channel *model.Channel) {
+	_ = p.updateAutomutingOnChannelCreated(channel)
+
+	if p.getConfiguration().UseSharedChannels && (channel.Type == model.ChannelTypeDirect || channel.Type == model.ChannelTypeGroup) {
```

Comment:
Let's use `channel.IsGroupOrDirect()` for clarity here.

Reply:
OK

Reply:
Done